src/components/singleArticle.jsx

```diff
@@ -28,10 +28,12 @@ const ArticleView = ({ article, className, bookmark }) => {
       </Link>
       <div className="article-meta">
         <div>
-          <span className="author-name">
-            <i className="zmdi zmdi-face mr-1" />
-            {article.author.username}
-          </span>
+          <Link to={`/${article.author.username}`}>
```

Comment:
@aaronsekisambu It would be better if you remove the span element and add it's class to the link element to avoid unnecessary HMTL elements.